main.go

```diff
@@ -1,48 +1,129 @@
 package main
 
 import (
-	"log"
-
+	"fmt"
 	"github.com/kradalby/opnsense-go/opnsense"
-	"github.com/satori/go.uuid"
+	//"github.com/satori/go.uuid"
+	"log"
 )
 
 func main() {
-	// c, err := opnsense.NewClient("http://localhost:8080", "", "", true)
+	c, err := opnsense.NewClient("http://192.168.45.128",
+		"98JfUoQ9QI8YUsUZ6h1eGG9PHaEFnBELf5ANlV/lTv5bWchyPOyZVV2a70rdDV0PvThBwzYYSLR0c/J1",
+		"MlMrlTkepR+CNuqpZ/dpBYcloiWogKPtgixM988MuPK7hYUtMFUsI1AQoHhfuqhEdEDPK0BXW8SN7V/0",
```

Comment:
Should probably remove this before merge :)

Reply:
Yes mybad, should be good now